with-remix/README.md

```diff
@@ -0,0 +1,34 @@
+# Welcome to Remix!
+
+- [Remix Docs](https://remix.run/docs)
```

Comment:
We have a template for the README, it can be found in the file TEMPLATE.md

Please have a look and adjust accordingly :)